packages/testing/src/TestUtils.ts

```diff
@@ -73,10 +71,7 @@ export function makeEncryptCallback({
  */
 export function makeDecryptCallback({
   secretKey,
-}: {
-  secretKey: Uint8Array
-  type: 'x25519'
-}): DecryptCallback {
+}: KiltEncryptionKeypair): DecryptCallback {
```

Comment:
Decrypting does not need the public key here, right? At least that is not what the function was expecting before.

Reply:
Yeah, but this here are just a bunch of testing utilities, it’s not too important here I think?

Reply:
Oh yeah, did not check what file it was 👍